.github/fixtures/test-bitbucket-integration/expected.md

```diff
@@ -17,4 +21,4 @@
 ### New Contributors
 * @orhun <orhun@archlinux.org> made their first contribution
 
-<!-- generated by -cliff -->
```

Comment:
Oh wow, any idea what happened here before?

Reply:
Both the template and expected output had the broken string.

https://github.com/orhun/git-cliff/pull/1090/files/a29045d02bcd67e5fcb4fe7d5904b5fffc1036ff#diff-4c1d6e9d77c9f3447022f6155d1d1fa6a722fa96e0e8194fe557a9931879403bL43